.github/workflows/pytest.yml

```diff
@@ -50,11 +50,11 @@ jobs:
           pip install --no-deps --editable .
           make docs-build
 
-      - name: Upload coverage
+      - name: Upload docs coverage artifact
         uses: actions/upload-artifact@v4
         with:
           name: coverage-docs
-          path: coverage.xml
+          path: ./*coverage*
```

Comment:
For all the coverage generating jobs, upload both the XML and DB versions of the coverage data.